changelog.d/5-internal/pr-3142

```diff
@@ -0,0 +1 @@
+Rust library `rusty-jwt-tools` upgraded to latest version
```

Comment:
Where is part (1/2) of this PR?  [Here](https://github.com/wireapp/wire-server/pull/2652)?

Do you have rusty-jwt-tools release notes that you can link to here?

Reply:
Part one: https://github.com/wireapp/wire-server/pull/3112

There are no release notes in [rusty-jwt-tools](https://github.com/wireapp/rusty-jwt-tools).